File: DSA_Java/stack.java
import java.util.Stack;
public class stack {
    public static void main(String[] args) {
        Stack<String> stack = new Stack<String>();
//        System.out.println(stack.empty());

        stack.push("Minecraft");
        stack.push("Skyrim");
        stack.push("DOOM");
        stack.push("Bordelands");
        stack.push("FFVII");

        String myFavGame = stack.pop();
        System.out.println(stack.peek());
        System.out.println(myFavGame);
        System.out.println(stack.search("Fallou76"));
        System.out.println(stack.search("DOOM"));

        System.out.println(stack);
    }
}

File: hello_算法_Python/stack.py
#初始化栈
stack:list[int] = []

#入栈
stack.append(1)
stack.append(3)
stack.append(2)
stack.append(5)
stack.append(4)

#访问栈顶元素
peek:int = stack[-1]

#出栈
pop:int = stack.pop()

#获取栈的长度
size:int = len(stack)

#判断是否为空
is_empty :bool = len(stack) == 0